src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -579,12 +579,30 @@ class ConsentGroupReview extends Component {
     }
   };
 
+  resetHandler () {
+    this.setState(prev => {
+      prev.resetIntCohorts = false;
+      prev.determination = this.resetIntCohortsDetermination();
+      return prev;
+    })
+  }
+
+  resetIntCohortsDetermination() {
+    return {
+      projectType: 900,
```

Comment:
Is there a list of what these numbers correspond to? This would be very confusing to a dev looking at this unless they knew the context (like me ... and I don't know the context). If the values can be constants defined somewhere with descriptive names, that would be more helpful.

Reply:
In order to prevent any confusion, I created an enum(ish) object that contains all possible determination types outputs coming from questionnaire workflow implementation. So now they are all together in one place. 

On the other hand, it wans't neccesary to inicialize any type of determination for this object, you will see them null on constructors and for reset purpouses.

Reply:
👍